tifeatures/factory.py

```diff
@@ -636,10 +637,12 @@ async def items(
                 properties=properties,
                 limit=limit,
                 offset=offset,
-                geom=geom_column,
+                geom=geom_column_name,
+                geometry_column=geom_column,
```

Comment:
I'm still not quite sure I understand why we need `geometry_column`, we should be able to get it using `geom_column` it feels we are duplicating the information and adding things in the factory/dependency for nothing!

Reply:
Because I'm lazy and that was easier to copy/paste than hunting down to make sure where I was expecting the name or the object.